avaframe/out3Plot/plotUtilsCfg.ini

```diff
@@ -61,18 +62,30 @@ unitFDV = $m^2s^{-1}$
 unitVx = $ms^{-1}$
 unitVy = $ms^{-1}$
 unitVz = $ms^{-1}$
+unitTA = $°$
 # threshold levels
 elevMaxppr = 100
 elevMaxpfd = 1
 elevMaxpfv = 10
+elevMaxpta = 40
 elevMaxP = 100
 elevMaxFV = 10
 elevMaxFD = 1
 elevMaxFDV = 10
+# contour levels
+contourLevelsppr = 1|3|5|10|25|50|100|250
+contourLevelspfd = 0.1|0.25|0.5|0.75|1
+contourLevelspfv = 0.5|1|5|10|25|50
+contourLevelspta = 10|15|20|25|30|35
+contourLevelsP = 1|3|5|10|25|50|100|250
+contourLevelsFV = 0.5|1|5|10|25|50
+contourLevelsFD = 0.1|0.25|0.5|0.75|1
+contourLevelsTA = 10|15|20|25|30|35
```

Comment:
good idea, but why not use it in the plotUtils?

Reply:
you mean use this for the levels to define the cmaps?

Reply:
Yes, this way they would be consistent. Or are these used in different ways?

Reply:
yes you need as many levels as colors for the cmaps...

Reply:
So it is basically the same, right? If you adjust this, one also has to adjust the plotUtils?